heartbeat/monitors/active/dialchain/builder.go

```diff
@@ -23,6 +23,7 @@ import (
 	"time"
 
 	"github.com/elastic/beats/heartbeat/monitors"
+	"github.com/elastic/beats/heartbeat/monitors/jobs"
```

Comment:
I'm wondering why the package is not call `job` instead of `jobs`?

Reply:
@ruflin the lame answer here is that it makes the code elsewhere confusings because we use the variable `jobs` a lot. I'm glad to change it, I guess elsewhere I could use `js := jobs.Job[]` instead of `jobs := job.Job[]`